src/stores/ContractsStore.ts

```diff
@@ -16,6 +16,7 @@ import transformObjectValues from 'helpers/transformObjectValues'
 class ContractsStore extends PersistableStore {
   connectedAccounts: { [account: string]: ContractSynchronizer } = {}
   currentBlock?: number
+  addressToTokenIds?: Promise<{ [address: string]: string[] } | undefined>
```

Comment:
Should we do both `?` and `undefined` here? Like I understand, that the value itself might be `undefined` and promise might return `undefined`, do we need both?

Reply:
I may be undefined or it may be Promise with undefined as I understood. So looks fine